VARIABLES.md

```diff
@@ -324,6 +324,14 @@ Default:  "{{sasl_protocol if sasl_protocol == 'kerberos' else 'none'}}"
 
 ***
 
+### zookeeper_log_dir
+
+Only use to customize directory Zookeeper writes log files to. Default location is /var/log/kafka. NOTE- zookeeper.log_path is deprecated.
```

Comment:
Only use to customize the directory that Zookeeper writes log files to. Default location is /var/log/kafka. NOTE- zookeeper.log_path is deprecated.